Pomodoro/app/src/main/res/layout/activity_main.xml

```diff
@@ -21,52 +21,61 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/startButtonText"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/changeTimerseekBar"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/countDownView"
-        />
+        app:layout_constraintVertical_bias="0.186" />
 
-    <Button
-        android:id="@+id/setTimeDurationTo30MinutesBt"
+    <TextView
+        android:id="@+id/numberOfRepetitionsTextView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/countDowntimeDuration30Minutes"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toStartOf="@id/setTimeDurationTo60MinutesBt"
+        android:text="@string/numberOfRepetitionsText"
+        app:layout_constraintEnd_toEndOf="@+id/numberOfRepetitionsEditText"
+        app:layout_constraintStart_toStartOf="@+id/numberOfRepetitionsEditText"
+        app:layout_constraintTop_toBottomOf="@+id/numberOfRepetitionsEditText" />
+
+    <EditText
+        android:id="@+id/numberOfRepetitionsEditText"
+        android:layout_width="28dp"
+        android:layout_height="41dp"
+        android:ems="10"
+        android:inputType="number"
+
+        app:layout_constraintBottom_toTopOf="@+id/changeTimerseekBar"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/startCountdownButton"
-        />
+        app:layout_constraintTop_toBottomOf="@+id/startCountdownButton" />
 
-    <Button
-        android:id="@+id/setTimeDurationTo60MinutesBt"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/countDowntimeDuration60Minutes"
+
+    <SeekBar
+        android:id="@+id/changeTimerseekBar"
+        android:layout_width="195dp"
+        android:layout_height="15dp"
```

Comment:
Ser du har lagt til min og max verdiene for seekbaren i main_activity.kt, men man kan legge til min og max som attributter her om man skulle ønske å "rydde" litt i koden :)